integration-tests/steps/drift.py

```diff
@@ -0,0 +1,76 @@
+from behave import *
+from helpers import get_cloudformation_stack_name
+
+import boto3
+
+from sceptre.plan.plan import SceptrePlan
+from sceptre.context import SceptreContext
+
+
+@given('a stack setting in stack "{stack_name}" has drifted')
```

Comment:
This step is very specific to log groups. Maybe this would be more accurate: 
```suggestion
@given('a log group configuration in stack "{stack_name}" has drifted')
```